Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/TestToolStartup.cs

```diff
@@ -267,6 +270,61 @@ private static LambdaFunction LoadLambdaFunction(LambdaConfigInfo configInfo, Lo
             return lambdaFunction;
         }
 
+        private static string DetermineLambdaContext(LocalLambdaOptions localLambdaOptions, CommandLineOptions commandOptions, string lambdaAssemblyDirectory, string lambdaProjectDirectory, RunConfiguration runConfiguration)
+        {
+            var lambdaContext = commandOptions.LambdaContext;
+
+            bool contextFileFound = false;
+            if (!string.IsNullOrEmpty(lambdaContext))
+            {
+                if (Path.IsPathFullyQualified(lambdaContext) && File.Exists(lambdaContext))
+                {
+                    runConfiguration.OutputWriter.WriteLine($"... Using context with from the file {lambdaContext}");
+                    lambdaContext = File.ReadAllText(lambdaContext);
+                    contextFileFound = true;
+                }
+                else
+                {
+                    // Look to see if the context value is a file in
+                    // * Directory with user Lambda assemblies.
+                    // * Lambda project directory
+                    // * Properties directory under the project directory. This is to make it easy to reconcile from the launchSettings.json file.
+                    // * Is a saved sample request from the web interface
+                    var possiblePaths = new[]
+                    {
+                        Path.Combine(lambdaAssemblyDirectory, lambdaContext),
+                        Path.Combine(lambdaProjectDirectory, lambdaContext),
+                        Path.Combine(lambdaProjectDirectory, "Properties", lambdaContext),
+                        Path.Combine(localLambdaOptions.GetPreferenceDirectory(false), new SampleRequestManager(localLambdaOptions.GetPreferenceDirectory(false)).GetSaveRequestRelativePath(lambdaContext))
+                    };
+                    foreach (var possiblePath in possiblePaths)
+                    {
+                        if (File.Exists(possiblePath))
+                        {
+                            runConfiguration.OutputWriter.WriteLine($"... Using payload with from the file {Path.GetFullPath(possiblePath)}");
```

Comment:
Output line says payload but this is for context. As this code needs to put in a common place there should be a output line in this method saying "Searching for X" and then in the common utility method inform the user where the file was found it at all found.